.github/workflows/ci.yml

```diff
@@ -23,8 +23,8 @@ jobs:
       - name: Get Composer Cache Directories
         id: composer-cache
         run: |
-          echo "files_cache=$(composer config cache-files-dir)" >> $GITHUB_OUTPUT
-          echo "vcs_cache=$(composer config cache-vcs-dir)" >> $GITHUB_OUTPUT
+          echo "files_cache=$(composer config cache-files-dir -q | grep '.cache/composer')" >> $GITHUB_OUTPUT
+          echo "vcs_cache=$(composer config cache-vcs-dir -q | grep '.cache/composer')" >> $GITHUB_OUTPUT
```

Comment:
Try using this high quality action: https://github.com/ramsey/composer-install

Reply:
Thanks for the suggestion. Would you like to open a PR to change only this part?

Reply:
All right.
That will cost you _One Million Dollars_.

Reply:
It didn't seem that complex. But you're the expert.

Reply:
Thank you https://github.com/vimeo/psalm/pull/10966